argoexec/requirements-sanity.in

```diff
@@ -0,0 +1,5 @@
+charmed_kubeflow_chisme
+juju~=2.9.0
```

Comment:
Why do we need this pin?

I don't think you need python-libjuju, pytest-operator, or ops for testing the rock.

Reply:
Will pin to <4